index.d.ts

```diff
@@ -8,11 +8,30 @@ import isUrl from 'is-url-superb';
 isUrl('https://sindresorhus.com');
 //=> true
 
-isUrl('//sindresorhus.com');
-//=> true
-
 isUrl('unicorn');
 //=> false
+
+// Enable lenient mode to automatically prepend `https://` to the input in case it has no protocol defined.
+isUrl('unicorn', {lenient: true});
+//=> true
 ```
 */
-export default function isUrl(url: string): boolean;
+export default function isUrl(
+	url: string,
+	options?: {
+		/**
+		 * Wether it should automatically prepend `https://` in case the input has no protocol defined.
+		 *
+		 * @example
+		 * isUrl('www.example.com');
+		 * //=> false
+		 *
+		 * isUrl('www.example.com', { lenient: true });
+		 * //=> true
+		 *
+		 * @default
+		 * false
+		 */
+		lenient: boolean;
+	}
```

Comment:
Can you follow this style? https://github.com/sindresorhus/typescript-definition-style-guide#documentation

Reply:
Sure! I've updated the definition file.